src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}
```
combine it with a comma